aten/src/ATen/native/transformers/cuda/flash_attn/flash_api.cpp

```diff
@@ -393,16 +393,16 @@ mha_fwd(const at::Tensor &q,         // batch_size x seqlen_q x num_heads x head
     bool is_sm8x = dprops->major == 8 && dprops->minor >= 0;
     bool is_sm90 = dprops->major == 9 && dprops->minor == 0;
     bool is_sm10x = dprops->major == 10 && dprops->minor >= 0;
-    bool is_sm120 = dprops->major == 12 && dprops->minor == 0;
-    TORCH_CHECK(is_sm120 || is_sm10x || is_sm90 || is_sm8x, "FlashAttention only supports Ampere GPUs or newer.");
+    bool is_sm120_or_sm121 = dprops->major == 12 && dprops->minor <= 1;
```

Comment:
Why do we need max versioning at all? Won't FlashAttn supports any GPU that can run the SMArch?